thrust/thrust/mr/memory_resource.h

```diff
@@ -204,7 +204,7 @@ _CCCL_HOST_DEVICE bool operator!=(const memory_resource<Pointer>& lhs, const mem
  *  \return a pointer to a global instance of \p MR.
  */
 template <typename MR>
-_CCCL_HOST MR* get_global_resource()
```

Comment:
Where is this getting used in a host/device function?

Reply:
https://github.com/NVIDIA/cccl/blob/f4d358a81c23a0171efcfc9f353cc238c9d7f95a/thrust/thrust/mr/allocator.h#L204-L206